packages/ui/src/components/popover/popover.vue

```diff
@@ -43,7 +43,9 @@
         class="acv-popover"
       >
         <AcvCard
-          class="acv-popover__content"
+          class="content"
+          background-color="white"
```

Comment:
mb, is it better to use css variables instead of props?

Reply:
Yes, but Card component is not fully implemented yet